packit_service/worker/reporting.py

```diff
@@ -165,8 +168,9 @@ def set_status(
         except gitlab.exceptions.GitlabCreateError as e:
             # Ignoring Gitlab 'enqueue' error
             # https://github.com/packit-service/packit-service/issues/741
-            if e.response_code == 403:
+            if e.response_code != 400:
                 # 403: No permissions to set status, falling back to comment
+                # 404: Commit has not been found, e.g. used target project on GitLab
```

Comment:
Do we know anything about other codes? I am thinking about something safer like this since we still document each known response code:

```python
if e.response_code not in {403, 404}:
```